lib/stream.js

```diff
@@ -9,22 +7,47 @@ if (!(config.get('nyplBaseUrl').endsWith('/'))) {
   nyplBaseUrl = config.get('nyplBaseUrl').concat('/')
 }
 console.log(nyplBaseUrl)
-var streamsClient = new NyplStreamsClient({ nyplDataApiClientBase: nyplBaseUrl})
+var streamsClient = new NyplStreamsClient({nyplDataApiClientBase: nyplBaseUrl})
 
 const isABib = process.env.RETRIEVAL_TYPE === 'bib'
 var log = isABib ? bunyan.createLogger({name: 'sierra-bib-retriever'}) : bunyan.createLogger({name: 'sierra-item-retriever'})
 
 // send data to kinesis Stream
 exports.streamPoster = (records, streamName) => {
-  return new Promise((resolve, reject) => {
-    streamsClient.write(streamName, records)
-      .then((resp) => {
-        log.info('Finished writing to stream ' + resp.Records.length)
-        resolve({sent: records.length, receieved: resp.Records.length})
-      })
-      .catch((error) => {
-        log.error('Error occurred while posting to kinesis')
-        reject(error)
-      })
-  })
+  return writeToStreamsClient(streamName, records)
+          .then(() => {
```

Comment:
Ah sorry I assumed standard would balk at the generous indentation there, but if standard is happy, I am.